src/mpid/ch4/netmod/ofi/subconfigure.m4

```diff
@@ -305,6 +305,17 @@ AM_COND_IF([BUILD_CH4_NETMOD_OFI],[
                                        [],
                                        [AC_MSG_ERROR(MPL_aligned_alloc is required to build OFI netmod)])
 
+    AC_ARG_ENABLE(ofi-domain,
+    [--enable-ofi-domain
+       Use fi_domain for vci contexts. The default is to use fi_contexts within a scalable endpoint.
```

Comment:
Is this help text correct?

Reply:
Good catch! It is incorrect. I'll update it.

Reply:
Sounds good. I think the rest of the PR looks fine after this update.